src/health_manager.py

```diff
@@ -13,140 +14,179 @@
 from ui import view, meters
 from ui_manager import ScreenObjects, is_visible
 
-pause_state = True
+_pause_state = True
 
 def get_pause_state():
-    global pause_state
```

Comment:
The global keyword is only necessary when modifying variables, not when you're just accessing them in a function